src/components/ErrorBoundary/ErrorBoundary.jsx

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'react';
+
+class ErrorBoundary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  componentDidCatch() {
```

Comment:
The `componentDidCatch` method should accept two parameters: `error` and `info`. These parameters provide details about the error and the component stack trace, which are useful for debugging.